src/library/autodj/autodjprocessor.h

```diff
@@ -154,6 +152,14 @@ class AutoDJProcessor : public QObject {
         ADJ_NOT_TWO_DECKS
     };
 
+    enum class TransitionMode {
+        AlignIntroOutroStart = 0,
+        AlignIntroOutroEnd = 1,
+        FixedFullTrack = 2,
+        FixedSkipSilence = 3,
+        FixedLoadAtCue = 4,
```

Comment:
What do you mean? How could AutoDJ use the preference option without interfering with any of the other modes' behavior?

Reply:
I mean the load point is already set in the deck preferences.
Here we override this behaviour. 
If one use the auto DJ as assistant it may interfere.

On the other hand having this easy accessible in the auto DJ Gui will be convenient if you switch from automated background music mode to  DJ-ing.

I think we need to re-think this once we have the other cases right.